streaming.md

```diff
@@ -3,24 +3,22 @@
 
-Matroska having a small overhead, it is well suited for storing music/videos on file servers without having a big impact on the bandwidth used. It doesn't require to load the index before playing (the index can be loaded only when seeking is requested the first time), so playback can start very quickly too.
+Matroska, having a small overhead, is well suited for storing music/videos on file servers without a big impact on the bandwidth used. Matroska does not require the index to be loaded before playing, which allows playback to start very quickly. The index can be loaded only when seeking is requested the first time.
 
-## Live Streaming
+## Livestreaming
 
-Live streaming is the equivalent of TV broadcasting on the internet. There are 2 families of servers for that. The RTP/RTSP ones and the HTTP servers. Matroska is not meant to be used over RTP. RTP already has timing and channel mechanisms that would wasted if doubled in Matroska. On the other hand live streaming of Matroska over HTTP (or any other plain protocol based on TCP) is very possible.
+Livestreaming is the equivalent of television broadcasting on the internet. There are 2 families of servers for livestreaming: RTP/RTSP and HTTP. Matroska is not meant to be used over RTP. RTP already has timing and channel mechanisms that would wasted if doubled in Matroska. Livestreaming of Matroska over HTTP (or any other plain protocol based on TCP) is possible.
 
-A live Matroska stream is different than a file, because it may have no known end (only when the client disconnects). For that the Segment MUST use the "unknown" size (all 1s in the size). The other option would be to concatenate Segments with known sizes one after the other. This solution allows a change of codec/resolution between each segment which can be useful in some cases (switch between 4:3 and 16:9 in some TV programs for example).
+A live Matroska stream is different from a file because it usually has no known end (only ending when the client disconnects). For this, the `Segment` MUST use the "unknown" size (all 1s in the size). Another option is to concatenate `Segment Elements` with known sizes, one after the other. This solution allows a change of codec/resolution between each segment. For example, this allows for a switch between 4:3 and 16:9 in a television program.
 
-The Segment(s) being continuous, certain elements like Meta Seek, Cues, Chapters, Attachments MUST NOT be used in this context.
+When `Segment Elements` are continuous, certain elements, like `MetaSeek`, `Cues Elements`, `Chapters Element`, and `Attachments Element`, MUST NOT be used.
```

Comment:
If you're using "certain elements", then I would repeat the word `Element` in the enumeration. Maybe this:

"…certain `Elements`, like `MetaSeek`, `Cues`, `Chapters` and `Attachments`,…"